frontend.sh

```diff
@@ -85,11 +85,14 @@ clean_and_install() {
 # Run tasks to build the project for distribution.
 build() {
   echo 'Building project…'
-  gulp clean
   gulp build
 
   if [ "$cli_flag" = "production" ]; then
+    echo 'Running additional build steps for on-demand and Nemo assets.'
     gulp scripts:ondemand
+    gulp styles:ondemand
+    gulp scripts:nemo
+    gulp styles:nemo
```

Comment:
Added these extra gulp steps that are currently done in the Jenkins build job, but seems like they should be done here instead. Once this is merged, I'll PR their removal from the Jenkins job.